### PR TITLE
[Doc] Update the component-description

### PR DESCRIPTION
**Pull request by myungjoo**
This document was way too obsolete. Update it with recent info.

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

### CODE REVIEW

File: Documentation/component-description.md [hunk 1]
```diff
@@ -35,22 +35,25 @@ In this page, we focus on the status of each elements. For requirements and desi
       - Invoke subplugin and let subplugin allocate output buffers.
       - Accept other/tensors.
     - TODO: Allow to manage synchronization policies.
-    - TODO: Recurrent models. The first target unit-test is LSTM.
   - Tensorflow-Lite (stable)
   - Custom (stable)
-  - Tensorflow (experimental)
-  - Other NNFW TBD (caffe2, caffe, ...)
+  - Tensorflow (stable)
+  - Tensorflow-lite (stable)
```
Comment (jaeyun-jung) [inline Documentation/component-description.md]: duplicated with line 38.

File: Documentation/component-description.md [hunk 2]
```diff
@@ -59,30 +62,51 @@ In this page, we focus on the status of each elements. For requirements and desi
...
+# API Support
+
+- C-API
+  - Main target is Tizen, but supports other OS as well.
+  - [Implementation](../api/capi) (stable, since Tizen 5.5 M2)
+- C#-API (.NET)
+  - Main target is Tizen, but supports other OS as well.
+  - [Implementation](https://github.com/Samsung/TizenFX/pull/940) (to be released for Tizen 5.5 M2)
+- JAVA-API (Android)
+  - [Android sample apps in nnstreamer-example.git](https://github.com/nnsuite/nnstreamer-example/tree/master/android/example_app) uses JAVA APIs to implement Android-nnstreamer apps.
```
Comment (jaeyun-jung) [inline Documentation/component-description.md]: how about linking the sample directory in this repo? actually all the android-application in nnstreamer-example does not use the APIs.

File: Documentation/component-description.md [hunk 3]
```diff
@@ -59,30 +62,51 @@ In this page, we focus on the status of each elements. For requirements and desi
...
+  - We will soon deploy nnstreamer via JCenter at "nnsuite" or "nnstreamer" repo of JCenter.
+    - com.samsung.android.nnstreamer
```
Comment (jaeyun-jung) [inline Documentation/component-description.md]: group-ID for release: com.nnsuite or com.samsung.android
which one is better? (#1642 to update build script)

File: Documentation/component-description.md (hunk 2 above)
Comment (myungjoo) [reply Documentation/component-description.md]: As the sample apps in nnstreamer-example are not using JAVA API, yet, I'll update this

File: Documentation/component-description.md (hunk 3 above)
Comment (myungjoo) [reply Documentation/component-description.md]: We have have voted a few minutes ago, let's define it "org.nnsuite.nnstreamer"

**Pull request merged**